tasks/build_tags.py

```diff
@@ -21,6 +21,7 @@
     "containerd",
     "cri",
     "docker",
+    "datadog.no_waf",
```

Comment:
`datadog.no_waf` is funny for a french speaker 🐶

Reply:
Woof woof !